setup.py

```diff
@@ -14,14 +14,14 @@ def get_packages(package):
 
 setup(
     name='zc_common',
-    version='0.3.7',
+    version='0.3.9b3',
```

Comment:
@birkholz Lets change this to a real version (including the url below) before merging. Also have to change the version in all the repos attached.